dialog/color_button.go

```diff
@@ -95,19 +94,21 @@ type colorButtonRenderer struct {
 func (r *colorButtonRenderer) Layout(size fyne.Size) {
 	r.rectangle.Move(fyne.NewPos(0, 0))
 	r.rectangle.Resize(size)
+	r.background.Resize(size)
 }
 
 func (r *colorButtonRenderer) MinSize() fyne.Size {
-	return r.rectangle.MinSize().Max(fyne.NewSize(32, 32))
+	return r.rectangle.MinSize().Max(fyne.NewSize(30, 30))
```

Comment:
I think this is unrelated, the checked pattern is not part of this PR

Reply:
The other pr is for the circular hatched pattern. This is to make the hatch pattern on the color buttons fit the box
As I mentioned in the discord, the code to add the pattern was already there, it just wasn't showing since it wasn't being resized to fit the area in the layout function (line 97)

Reply:
> The other pr is for the circular hatched pattern. This is to make the hatch pattern on the color buttons fit the box As I mentioned in the discord, the code to add the pattern was already there, it just wasn't showing since it wasn't being resized to fit the area in the layout function (line 97)

Aha I understand, sorry I missed that before.
Would it not be a smaller change to set the squares of the hatching to 8pt instead of 10? Then the UI layout remains the same but the bug fix focuses on the presentation, which is what you are fixing...

Reply:
That is also an option, though if this is done the advanced picker needs to be scaled so the preview won't be cut off. But I can do that and then just leave it like that and have it be fixed in the other pr where I can control the size of the preview directly

Reply:
Not quite sure I follow, sorry - are you saying there is an element of the advanced is aligned to the 10pt multiple?

Reply:
Nevermind, I thought it fit perfectly on this, I will change it to be 8
![image](https://user-images.githubusercontent.com/16857861/188216178-24cf1a07-7a55-457f-9000-8ed1d91548f2.png)